src/main/scala/io/gatling/build/automated/GatlingAutomatedScalafixPlugin.scala

```diff
@@ -25,16 +25,24 @@ import sbt._
 import sbt.Keys._
 
 object GatlingAutomatedScalafixPlugin extends AutoPlugin {
+
   override def requires: Plugins = ScalafixPlugin && GatlingBuildConfigPlugin
 
+  override def globalSettings: Seq[Def.Setting[_]] = Seq(
+    scalafixOnCompile := true
+  )
+
   trait GatlingAutomatedScalafixKeys {
     val gatlingScalafixCheck = taskKey[Unit]("Check that gatling scalafix rules have been applied")
 
     def automateScalafixBeforeCompile(configurations: Configuration*): Seq[Setting[_]] =
       configurations.foldLeft(List.empty[Setting[_]]) {
         _ ++ inConfig(_)(
           Seq(
-            compile := compile.dependsOn(scalafix.toTask("").dependsOn(scalafixWriteConfigFile)).value
+            compile := {
+              if (scalafixOnCompile.value) compile.dependsOn(scalafix.toTask("").dependsOn(scalafixWriteConfigFile)).value
+              else compile.value
+            }
```

Comment:
When multilined, please use braced branches.

```suggestion
            compile := {
              if (scalafixOnCompile.value) {
                compile.dependsOn(scalafix.toTask("").dependsOn(scalafixWriteConfigFile)).value
              } else {
                compile.value
              }
            }
```

Reply:
Forgotten?